src/toolchain/toolchain.ts

```diff
@@ -35,29 +35,37 @@ export class SwiftToolchain {
         public swiftVersion: Version,
         public toolchainPath?: string,
         public developerDir?: string,
+        public sdkroot?: string,
```

Comment:
Also is it necessary to store the `sdkroot` if it is only used in `getXCTestPath`?

Reply:
I guess we will definitely use it elsewhere, eg. in pre-5.7 compile flags using with script mode.

It’s a question if we should store `SDKROOT` for Darwin too because it also makes some sense there.

Reply:
Ok we should store `SDKROOT` if we are going to use it in the future. Not sure we need it for Darwin though

Reply:
Could you please be explicit if you would like me to make that change?  I don't have a strong opinion on whether we collect and store the value or not.

Reply:
We should continue to store sdkroot. So no change needed here